acos_client/v21/action.py

```diff
@@ -35,3 +35,16 @@ def reload(self, write_memory=False, **kwargs):
         write_memory = 1 if write_memory else 0
         return self._post("system.action.reload",
                           params={"write_memory": write_memory}, **kwargs)
+
+    def write_active(self, partitions=[], **kwargs):
```

Comment:
I'm not feeling this array interface. As a standalone library, it's awkward for the common case. For an array, a list comprehension by the caller is similar in complexity.